ui/pages/confirmations/components/confirm/info/shared/transaction-details/transaction-details.tsx

```diff
@@ -118,13 +118,13 @@ const PaymasterRow = () => {
 
 export const TransactionDetails = () => {
   return (
-    <>
+    <ConfirmInfoSection>
       <ConfirmInfoSection>
         <OriginRow />
         <RecipientRow />
         <MethodDataRow />
       </ConfirmInfoSection>
       <PaymasterRow />
-    </>
+    </ConfirmInfoSection>
```

Comment:
The paymaster row is shown in a separate section given the paymaster was a unique concept.

Meaning we shouldn't need a section around everything, but just the core rows as we see on line 126.